ui/apps/platform/src/Containers/ComplianceEnhanced/ClusterCompliance/ScanConfigs/Table/ScanConfigsTablePage.tsx

```diff
@@ -204,9 +208,9 @@ function ScanConfigsTablePage({
                     <TableComposable>
                         <Thead noWrap>
                             <Tr>
-                                <Th sort={getSortParams('Name')}>Name</Th>
+                                <Th sort={getSortParams('Compliance Scan Config Name')}>Name</Th>
```

Comment:
Should we keep this consistent with the rest of the publicly-facing terminology and have the heading read "Compliance Scan **Schedule** Name" ?

Reply:
Although maybe too late for the less visible response type name to change, Van makes a good point about this which appears in query string of page address. Requires coordination with backend, of course.

Reply:
I don't think we have to keep the names in code always in sync with every change in user-facing text.

I have long experience with content management systems, which are founded on the assumption that the end-users can change text as much as they want, while the underlying code for those systems will remain unchanged from its creation until the heat-death of the universe.

> “No man ever steps in the same river twice, for it's not the same river and he's not the same man.”
― Heraclitus